spec/features/staff/fund_managers_can_edit_a_fund_spec.rb

```diff
@@ -12,14 +12,14 @@
 
   context "when the user is a fund_manager" do
     before do
-      authenticate!(user: build_stubbed(:fund_manager, organisation: organisation))
+      authenticate!(user: create(:fund_manager, organisation: organisation))
```

Comment:
We used a `let` in spec/features/staff/fund_managers_can_create_an_organisation_spec.rb but it was in the `before` for this one, which I like better – could we do one for consistancy?